config/settings.yml

```diff
@@ -6,6 +6,7 @@
       :is_global_admin: false
     :inventory_object_refresh: true
     :allow_targeted_refresh: true
+    :parallel_thread_limit: 0
```

Comment:
Just for understanding, 0 means no parallelization (single 1 thread) or disable part of code providing parallel refresh functionality?

Reply:
0 means no parallelization.

Reply:
Thanks, merging.